cleanlab/count.py

```diff
@@ -101,7 +101,9 @@ def num_label_issues(
           1. As we add more label and data quality scoring functions in :py:mod:`cleanlab.rank`, this approach will always work.
           2. If you have a custom score to rank your data by label quality and you just need to know the cut-off of likely label issues.
 
-       TL;DR: use this method to get the most accurate estimate of number of label issues when you don't need the indices of the label issues.
+       TL;DR: Use this method to get the most accurate estimate of number of label issues when you don't need the indices of the label issues.
+       Note: ``'off_diagonal'`` may sometimes underestimate issues for low number of classes but you can always change frac_noise > 1 or get :py:func:`rank.get_label_quality_scores
+      <cleanlab.rank.get_label_quality_scores>` and choose your own cutoff.
```

Comment:
I believe this comment should be intended for `filter.find_label_issues()` docstring

Reply:
The analogous comment for this docstring here would be something like:

> Note: ``'off_diagonal'`` may sometimes underestimate issues for data with few classes, so consider using ``off_diagonal_calibrated`` instead if your data has < 10 classes.

(I just made up the number 10 here, you can put whatever number seems good from your benchmarks as a suggestion)